.github/workflows/renovate.yml

```diff
@@ -22,6 +16,8 @@ jobs:
         uses: renovatebot/github-action@v40.1.12
         with:
           configurationFile: renovate-config.json
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.PCCIBOT_PAT_REPO_PUBLIC_REPO }}
```

Comment:
Why is CI working in https://github.com/voxpupuli/puppet-prometheus/pull/776 when the new token isn't used yet? :thinking:

Reply:
I ran the task manually on the branch which uses the token